modal/functions.py

```diff
@@ -1460,8 +1461,8 @@ async def _gather(*function_calls: _FunctionCall):
 gather = synchronize_api(_gather)
 
 
-_current_input_id = ContextVar("_current_input_id")
-_current_function_call_id = ContextVar("_current_function_call_id")
+_current_input_id: ContextVar = ContextVar("_current_input_id")
+_current_function_call_id: ContextVar = ContextVar("_current_function_call_id")
```

Comment:
This type annotation conflicts with our auto-generated `inv type-stubs` machinery, which generates type stubs for the client library. That's it was omitted. Not sure what the difference between omitting and having it is here?

Reply:
I'm also less clear why this is raised; I usually see mypy requiring a declaration over compound data structures like lists and dictionaries (say needing `List[bool]` and `Dict[str, int]` vs `List` and `Dict`). Let me try getting `inv mypy` to pass locally and see what else I find.

Reply:
An explanation as for why this makes a difference:

We use synchronicity's type-stub generation to emit .pyi-files for these files in order to get type info for the runtime-generated blocking/async interfaces of `Function` and not just the internal `_Function` type.

To do that, synchronicity reads annotations, conditionally transforms them, and then re-emits them in the output .pyi-files. In the case of module globals they are only emitted in the .pyi-file if they are explicitly annotated in the original file, so without this annotation here these variables aren't even included in the .pyi-files

Reply:
Got it! My understanding here is to incorporate these changes, let me know otherwise.

That being said, I do see these in my `.pyi` file (after running `inv protoc` and `inv type-stubs`:

```python
_current_input_id: _contextvars.ContextVar

_current_function_call_id: _contextvars.ContextVar
```

Reply:
Those should be fine